Tests/CSharp/src/Tests/AssetSystemTests.cs

```diff
@@ -234,7 +234,7 @@ public static void CreateAssetCollectionNoSpaceTest()
             GetAssetCollectionByName(assetSystem, testAssetCollectionName, out var assetCollection);
 
             Assert.AreEqual(assetCollection.Name, testAssetCollectionName);
-            Assert.IsTrue(assetCollection.SpaceIds.IsEmpty());
```

Comment:
Again, this assertion was here specifically to ensure we could create AssetCollections with no space IDs.

Reply:
Please see the above notes regarding the decisions made in the engineering sync